apps/api/src/app/widgets/usecases/initialize-session/initialize-session.usecase.ts

```diff
@@ -26,7 +34,23 @@ export class InitializeSession {
       throw new ApiException('Please provide a valid app identifier');
     }
 
-    if (environment.widget.notificationCenterEncryption) {
+    const response = await this.getDecryptedIntegrations.execute(
+      GetDecryptedIntegrationsCommand.create({
+        findOne: true,
+        active: true,
+        environmentId: environment._id,
+        organizationId: environment._organizationId,
+        userId: command.subscriberId,
+      })
+    );
+
+    const item = Array.isArray(response) ? response[0] : response;
```

Comment:
Not sure if I am missing any detail but this bit of code can lead to bugs. If we now make the In App as a provider, that means it could be not enabled. And at the very same time a different channel provider enabled. So we should check that from the decrypted active integrations we have the In App one enabled.

Reply:
Yeah you found it at the same time as I did ;)